webpack/components/extensions/HostDetails/Cards/HostCollectionsCard.js

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from 'react';
+import {
+  Badge,
+  Card,
+  CardHeader,
+  CardTitle,
+  CardBody,
+  Dropdown,
+  ExpandableSection,
+  KebabToggle,
+  Flex,
+  FlexItem,
+  GridItem,
+  DropdownItem,
+} from '@patternfly/react-core';
+
+import { translate as __ } from 'foremanReact/common/I18n';
+import { propsToCamelCase } from 'foremanReact/common/helpers';
+import PropTypes from 'prop-types';
+import { useSet } from '../../../Table/TableHooks';
+
+const HostCollectionsDetails = ({
+  hostCollections,
+}) => {
+  const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+  const toggleBulkAction = () => setIsDropdownOpen(prev => !prev);
+
+  const expandedHostCollections = useSet([]);
+  const openAddHostCollectionsModal = () => {}; // TODO: implement
+  const openRemoveHostCollectionsModal = () => {}; // TODO: implement
+
+  const dropdownItems = [
+    <DropdownItem
+      aria-label="add_host_to_collections"
+      key="add_host_to_collections"
+      component="button"
+      isDisabled
+      onClick={openAddHostCollectionsModal}
+    >
+      {__('Add host to collections')}
+    </DropdownItem>,
+    <DropdownItem
+      aria-label="remove_host_from_collections"
+      key="remove_host_from_collections"
+      component="button"
+      isDisabled
+      onClick={openRemoveHostCollectionsModal}
+    >
+      {__('Remove host from collections')}
+    </DropdownItem>,
+  ];
+
+  return (
+    <GridItem rowSpan={1} md={6} lg={4} xl2={3} >
+      <Card isHoverable>
+        <CardHeader>
+          <Flex
+            alignItems={{ default: 'alignItemsCenter' }}
+            justifyContent={{ default: 'justifyContentSpaceBetween' }}
+            style={{ width: '100%' }}
+          >
+            <FlexItem>
+              <Flex
+                alignItems={{ default: 'alignItemsCenter' }}
+                justifyContent={{ default: 'justifyContentSpaceBetween' }}
+              >
+                <FlexItem>
+                  <CardTitle>{__('Host collections')}</CardTitle>
+                </FlexItem>
+                <FlexItem>
+                  {!!hostCollections?.length && <Badge isRead>{hostCollections.length}</Badge>}
```

Comment:
Learned something new about this :) Didn't know you could cast to bool with strings etc

Reply:
Yeah, `0` is falsey in JavaScript so using it with `length` is handy :)